.travis.yml

```diff
@@ -3,26 +3,28 @@ os: linux
 dist: xenial
 language: ruby
 cache: bundler
-before_install:
-  - bundle -v
-  - rm -f Gemfile.lock
-  - "# Update system gems if requested. This is useful to temporarily workaround troubles in the test runner"
-  - "# See https://github.com/puppetlabs/pdk-templates/commit/705154d5c437796b821691b707156e1b056d244f for an example of how this was used"
-  - "# Ignore exit code of SIGPIPE'd yes to not fail with shell's pipefail set"
-  - '[ -z "$RUBYGEMS_VERSION" ] || (yes || true) | gem update --system $RUBYGEMS_VERSION'
-  - gem --version
-  - bundle -v
+# before_install:
+#   - bundle -v
+#   - rm -f Gemfile.lock
+#   - "# Update system gems if requested. This is useful to temporarily workaround troubles in the test runner"
+#   - "# See https://github.com/puppetlabs/pdk-templates/commit/705154d5c437796b821691b707156e1b056d244f for an example of how this was used"
+#   - "# Ignore exit code of SIGPIPE'd yes to not fail with shell's pipefail set"
+#   - '[ -z "$RUBYGEMS_VERSION" ] || (yes || true) | gem update --system $RUBYGEMS_VERSION'
+#   - gem --version
+#   - bundle -v
 before_script:
   - wget https://apt.puppet.com/puppet-tools-release-xenial.deb
   - sudo dpkg -i puppet-tools-release-xenial.deb
   - sudo apt-get update
   - sudo apt-get install pdk -y
 script:
-  - 'pdk validate'
-bundler_args: --without system_tests
-rvm:
-  - 2.7.6
+  - 'pdk $CHECK'
+# bundler_args: --without system_tests
+# rvm:
+#   - 2.5.7
+#   - 2.7.6
```

Comment:
This other section can be deleted now :) 

P.S: I'm curious if the below GitHub suggestion will do the job:

```suggestion

```